app/models/ecf_participant_validation_data.rb

```diff
@@ -1,9 +1,19 @@
 # frozen_string_literal: true
 
 class ECFParticipantValidationData < ApplicationRecord
-  belongs_to :participant_profile, class_name: "ParticipantProfile::ECF", touch: true
+  belongs_to :participant_profile, class_name: "ParticipantProfile", touch: true
+  validate :belongs_to_ecf_participant
```

Comment:
Do we lose anything here by making the relationship generic and validating rather than prescribing types? 🤔

Reply:
I can't think of anything right now but we can fix it later if something arises.